docs-source/content/userguide/managing-domains/accessing-the-domain/admin-console.md

```diff
@@ -1,6 +1,115 @@
+You connect to the Remote Console using a web browser and, when prompted, supply the WebLogic Server login credentials
+along with the URL of the WebLogic Server Administration Server's administration port to which you want to connect.
+
+**Note**:  An Administration Server administration port typically is the same as its default port unless either an SSL port or an administration port is configured and enabled.
+
+### Setup
+
+To set up access to WebLogic Server domains running in Kubernetes using the Remote Console:
+
+1. Install, configure, and start the Remote Console according to these [instructions](https://github.com/oracle/weblogic-remote-console/blob/master/site/install_config.md).
+
+   **NOTE**: These instructions assume that you are installing and running the Remote Console Java program externally to your Kubernetes cluster.
+
+1. When you first connect your browser to the Remote Console, which is at `http://localhost:8012` by default, the console will prompt you with a login dialog for a WebLogic Server Administration Server URL. To give the Remote Console access to an Administration Server running in Kubernetes, you can:
+
+   * Use an [Administration Server `NodePort`](#use-an-administration-server-nodeport).
+
+   * Deploy a load balancer with [ingress path routing rules](#configure-ingress-path-routing-rules).
+
+
+#### Use an Administration Server `NodePort`
+
+For the Remote Console to connect to the Kubernetes WebLogic Server Administration Server’s `NodePort`, use the URL:
+
+```
+http://hostname:adminserver-NodePort/
+```
+
+The `adminserver-NodePort` is the port number of the Administration Server outside the Kubernetes cluster.
+For information about the `NodePort` Service on an Administration Server, see the [Domain resource](https://github.com/oracle/weblogic-kubernetes-operator/blob/master/docs/domains/Domain.md) document.
+
+{{% notice warning %}}
+Exposing administrative, RMI, or T3 capable channels using a Kubernetes `NodePort`
+can create an insecure configuration. In general, only HTTP protocols should be made available externally and this exposure
+is usually accomplished by setting up an external load balancer that can access internal (non-`NodePort`) services.
+For more information, see [T3 channels]({{<relref "/security/domain-security/weblogic-channels#weblogic-t3-channels">}}).
+{{% /notice %}}
+
+#### Configure ingress path routing rules
+
+1. Configure an ingress path routing rule. For information about ingresses, see the [Ingress]({{< relref "/userguide/managing-domains/ingress/_index.md" >}}) documentation.
+
+   For an example, see the following `path-routing` YAML file for a Traefik load balancer:
+
+   ```yaml
+   apiVersion: traefik.containo.us/v1alpha1
+   kind: IngressRoute
+   metadata:
+     annotations:
+       kubernetes.io/ingress.class: traefik
+     name: traefik-pathrouting-1
+     namespace: weblogic-domain
+   spec:
+     routes:
+     - kind: Rule
+       match: PathPrefix(`/`)
```

Comment:
@maggiehe00 Any news about using '/management/' instead?

Reply:
/management does not work

Reply:
Argh. 

@lfeigen Any advice? Sadly, "/" captures all traffic.

Reply:
So, I missed something.  Why doesn't /management work?

Reply:
@moreaut ^^^